### DISCUSSION TITLE
Inquiry about Dockerized Version of Copacetic

### DISCUSSION THREAD

**Discussion by AshkanRafiee** [project-copacetic/copacetic#419] [General]
### What is your question?

Hello Copacetic Team,

I hope this message finds you well. I am currently exploring the use of Copacetic in my projects and was wondering about its compatibility with Docker.

Is there an official Dockerized version of Copacetic available? If not, are there any plans to release it in the near future?

Having an Official Dockerized version would greatly enhance the ease of deployment and integration in various environments, and I believe it would be a valuable addition for users like myself who rely heavily on Docker for containerization.

Thank you for your time and effort in developing Copacetic. I look forward to your response and any updates on this matter.

Best regards,

**Comment by sozercan** [comment]
hi @AshkanRafiee! we have an image that is used for the copa github action. 
image: https://github.com/project-copacetic/copacetic/pkgs/container/copa-action
source: https://github.com/project-copacetic/copa-action
we don't support this outside of GH Action at this time, so we don't have any docs around this.

@toddysm used this with success recently and might be able to share some learnings.
